### PR TITLE
Small update to artifacts docs examples

### PR DESCRIPTION
**Pull request by zhen0**
Updates the docs to include valid markdown examples that can be properly rendered in the UI. 

Comments / feedback and ideas on the links section are particularly welcome - I expect there is a way we can add the time and formatting and still make it valid markdown.  But given we have a release coming an example that formats correctly feels more important. 

### Example

Previous markdown docs example rendered in the UI:

image

Updated version:

image

Previous link and updated link in the UI:
image

### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [ ] This pull request includes tests or only affects documentation.
- [ ] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`, `docs`.

### CODE REVIEW

**Comment by serinamarie** [comment]
I think it might be good to rename "highest_churn_possibility" to "_probability" but open to thoughts.

File: docs/concepts/artifacts.md [hunk 1]
```diff
@@ -33,23 +33,17 @@ import pendulum
 
 @task
 def my_first_task():
-    now = pendulum.now('UTC').to_datetime_string()
-    data_variability = 1764 ** 0.5
-    if data_variability >= 42:
         create_link_artifact(
             key="irregular-data",
-            link=f"https://nyc3.digitaloceanspaces.com/my-bucket-name/highly_variable_data_{now}.csv",
+            link=f"https://nyc3.digitaloceanspaces.com/my-bucket-name/highly_variable_data.csv",
```
Comment (serinamarie) [inline docs/concepts/artifacts.md:38]:
```suggestion
            link="https://nyc3.digitaloceanspaces.com/my-bucket-name/highly_variable_data.csv",
```

File: docs/concepts/artifacts.md [hunk 2]
```diff
@@ -33,23 +33,17 @@ import pendulum
...
 
 @task
 def my_second_task():
-    now = pendulum.now('UTC').to_datetime_string()
-    prediction_accuracy = 0.42
-    if prediction_accuracy < 0.50:
         create_link_artifact(
             key="irregular-data",
-            link=f"https://nyc3.digitaloceanspaces.com/my-bucket-name/low_pred_data_{now}.csv",
+            link=f"https://nyc3.digitaloceanspaces.com/my-bucket-name/low_pred_data.csv",
```
Comment (serinamarie) [inline docs/concepts/artifacts.md:46]:
```suggestion
            link="https://nyc3.digitaloceanspaces.com/my-bucket-name/low_pred_data.csv",
```

File: docs/concepts/artifacts.md [hunk 3]
```diff
@@ -96,47 +90,45 @@ from prefect import flow, task
 from prefect.artifacts import create_markdown_artifact
 
 @task
-def my_task():
+def markdown_task():
     na_revenue = 500000
-    markdown_report = """
-        # Sales Report
+    markdown_report = f"""# Sales Report
```
Comment (cicdw) [inline docs/concepts/artifacts.md:95]: next necessarily needed here, but just as an FYI you can use `import textwrap; textwrap.dedent()` on large blocks of text like this to remove all leading whitespace from every line.  Sometimes makes it easier to manage strings like this

**Review by cicdw — APPROVED** [review]

File: docs/concepts/artifacts.md (hunk 3 above)
Comment (zhen0) [reply docs/concepts/artifacts.md:95]: Thank you!

**Pull request merged**